client/axios-instance.js

```diff
@@ -0,0 +1,114 @@
+const axios = require('axios')
+const jwtDecode = require('jwt-decode')
+const utils = require('./utils')
+
+// env consts
+const BASE_SERVER_URL = process.env.BASE_SERVER_URL || 'http://127.0.0.1:8888/'
+const SERVER_USER_EMAIL = process.env.SERVER_USER_PASSWORD
+const SERVER_USER_PASSWORD = process.env.SERVER_USER_PASSWORD
+
+let accessToken
+let refreshToken
+
+// init axios instance with base url
+const instance = axios.create({
+  baseURL: BASE_SERVER_URL,
+  timeout: 1000
+})
+
+const isTokenAlive = (token) => {
+  if (!token) return false
+
+  let decoded = jwtDecode(token)
+  return decoded.exp > (new Date().getTime() / 1000)
+}
```

Comment:
Do we really need to decode the token on the client side? First of all, I am not sure that it will work with real (not mocked) secret for the tokens' encoding, because you don't know it on the client side. Also now you can distinguish if the access token is expired by the error code.